src/editor/components/components/AddLayerPanel/AddLayerPanel.component.jsx

```diff
@@ -145,7 +145,11 @@ const createEntity = (mixinId) => {
   }
   const newEntityObject = {
     mixin: mixinId,
-    components: {}
+    components: {
+      'accent-color': {
+        index: -1
+      }
+    }
```

Comment:
I wouldn't add the component by default, that's not needed unless you use it. I would create a button in the Sidebar for example "customize color" (for mixins that makes sense) that adds the component to the entity. You can do that with `AFRAME.INSPECTOR.execute('componentadd', {entity: entity, component: 'accent-color'})`